iTechArt.Repositories.EntityFramework/Interfaces/IDbContext.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Data.Entity;
+using System.Data.Entity.Infrastructure;
+using System.Threading.Tasks;
+
+namespace iTechArt.Repositories.EntityFramework.Interfaces
+{
+    public interface IDbContext
+    {
+        IDbSet<TEntity> Set<TEntity>() where TEntity : class;
```

Comment:
We can use default DbContext implementation: DbSet<TEntity> Set<TEntity>() and remove explicit implementation from our SurveyContext.